bin/update-data.py

```diff
@@ -97,39 +97,36 @@ def load_ecdc_diff_data(result, dates):
   dt = latest_day - first_day
   if verbosity > 0:
     print('Reading ECDC daily diff data ...')
-  diff_data = None
```

Comment:
You're right, it's unnecessary. But I like to keep it for clarity.

Reply:
Okay, take my pull request, and put it in the Tonne :-)

Reply:
or must i delete this request? but how?